### PR TITLE
GEOMESA-1175 Doc fixes for 1.2.2 release.

### PR DESCRIPTION
**Pull request by mdzimmerman**
* Update settings in common.py.
* Remove references to vecmath.
* Update Kafka tutorial with feature listener and load tester.

Signed-off-by: Matthew Zimmerman <matt.zimmerman@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
looks good - should we keep this open for more fixes or just merge it?

**Comment by mdzimmerman** [comment]
Yeah, I would leave it open for now in case there are other docs fixes to be made.

File: docs/user/installation_and_configuration.rst [hunk 1]
```diff
@@ -459,3 +475,12 @@ is printed out when you start GeoServer::
     ----------------------------------
     - GEOSERVER_DATA_DIR: /opt/devel/install/geoserver-data-dir
     ----------------------------------
+
+Note about Apache Commons Collections
```
Comment (jnh5y) [inline docs/user/installation_and_configuration.rst]: @mdzimmerman, I tried to help;)  Feel free to fix the formatting if it is messed up or can be improved.

File: docs/user/installation_and_configuration.rst [hunk 2]
```diff
@@ -459,3 +475,12 @@ is printed out when you start GeoServer::
...
+-------------------------------------
+
+Version 3.2.1 and earlier of the Apache Commons Collections library have a CVSS 10.0 vulnerability.  Read more `here
+https://commons.apache.org/proper/commons-collections/security-reports.html`__.
+
+Note that Accumulo 1.6.5 is the first version of Accumulo which addresses this security concern.
```
Comment (cne1x) [inline docs/user/installation_and_configuration.rst]: Have we updated GeoMesa to this version of Accumulo?

File: geomesa-tools/README.md [hunk 3]
```diff
@@ -9,22 +9,26 @@ To begin using the command line tools, first build the full GeoMesa project from
...
     
-    cd geomesa-${version}
+    cd geomesa-tools-${version}
 
 GeoMesa Tools relies on a GEOMESA_HOME environment variable. Running
     
     . bin/geomesa configure
 
-with the `. ` prefix will set this for you, add $GEOMESA_HOME/bin to your PATH, and source your new environment variables in your current shell session.
+with the `. ` prefix will set this for you, add $GEOMESA_HOME/bin to your PATH, and source your new environment variables in your current shell session. Now you should be able to use GeoMesa from any directory on your computer.
```
Comment (cne1x) [inline geomesa-tools/README.md]: Should we use the more explicit ```source``` instead of ```.```?

File: geomesa-tools/README.md [hunk 4]
```diff
@@ -9,22 +9,26 @@ To begin using the command line tools, first build the full GeoMesa project from
...
 
-Now, you should be able to use GeoMesa from any directory on your computer. To test, `cd` to a different directory and run:
+Note: the tools will read the ACCUMULO_HOME and HADOOP_HOME environment variables to load the appropriate JAR files for Hadoop, Accumulo, Zookeeper, and Thrift. If possible, we recommend installing the tools on the Accumulo master server, as you may also need various configuration files from Hadoop/Accumulo in order to run certain commands. Use the ``geomesa classpath`` command in order to see what JARs are being used.
```
Comment (cne1x) [inline geomesa-tools/README.md]: [picky] Capitalize the first letter of a sentence following a colon?

File: geomesa-tools/README.md [hunk 5]
```diff
@@ -126,19 +130,12 @@ Due to licensing restrictions, a number of necessary dependencies required for r
...
     </dependency>
-    <dependency>
-        <groupId>java3d</groupId>
-        <artifactId>vecmath</artifactId>
-        <version>1.3.2</version>
-    </dependency>
 
 
-To install, you can either locate the jar files or run the two following included scripts which will attempt to wget and install the jars.
+To install, you can either locate the jar files or run the following included script which will attempt to wget and install the jars.
```
Comment (cne1x) [inline geomesa-tools/README.md]: The Oracle convention appears to be to capitalize "JAR".  Do we have a convention?

Comment (elahrvivaz) [inline geomesa-tools/README.md]: Hmmm, I prefer the lower-case... kind of like how Internet is finally now internet.

Comment (mdzimmerman) [inline geomesa-tools/README.md]: I agree we should standardize on "JAR", and we are using "JAR" exclusively in the RST docs. The READMEs, however, are more inconsistent on this. I think I used I followed the existing usage in an attempt to tread lightly, but I will fix it.

File: geomesa-tools/README.md (hunk 4 above)
Comment (mdzimmerman) [inline geomesa-tools/README.md]: You are right! Even Chicago Manual of Style recommends capitalizing after the colon if at least two complete sentences follow it (http://www.apvschicago.com/2011/04/capitalization-after-colons.html). Good to know!

File: geomesa-tools/README.md (hunk 3 above)
Comment (mdzimmerman) [inline geomesa-tools/README.md]: Yeah, that's probably a better idea.

File: docs/user/installation_and_configuration.rst (hunk 2 above)
Comment (jnh5y) [inline docs/user/installation_and_configuration.rst]: Nope, but versions of Accumulo seem to be 'binary' compatible.

Would it make sense to spell out the implications?  Or would that confuse matters?

File: geomesa-tools/README.md (hunk 5 above)
Comment (mdzimmerman) [inline geomesa-tools/README.md]: Yeah, but JAR is an acronym, and while there is some tendency for acronyms to become lowercase as they enter popular usage (e.g. RADAR -> radar), it tends not to happen if they displace preexisting words.

File: docs/user/installation_and_configuration.rst (hunk 2 above)
Comment (cne1x) [inline docs/user/installation_and_configuration.rst]: If they're binary compatible, then I would suggest we upgrade to 1.6.5 in our POM.  That way, we're sending a consistent message (at least on this single, small point).  It's not a democracy, but if it were, that's how I'd vote.

Comment (elahrvivaz) [inline docs/user/installation_and_configuration.rst]: @jnh5y would that impact our CQs?

**Pull request merged**